modules/store/spec/runtime_checks.spec.ts

```diff
@@ -339,6 +339,12 @@ function reducerWithBugs(state: any = {}, action: any) {
         invalid: new Date(),
       };
 
+    case '@ngrx ' + ErrorTypes.UnserializableState:
```

Comment:
Do we need these added cases here?

Reply:
@timdeschryver I found the two `it('should not throw for NgRx actions'...` tests to were passing by default . Because no cases in the reducer match `'@ngrx ' +  ErrorTypes.UnserializableState` or `'@ngrx ' + ErrorTypes.MutateAction` they both fall through to the default which returns the initial state and passes by default. But after fixing that I realized that it should be testing ngrx actions with Unserializable Actions not Unserializable State (as that's the exception granted to ngrx actions). So I changed the test but forgot to change the additional case until you pointed this out thanks!